federation/tests/resources/https_spiffe_server.go

```diff
@@ -0,0 +1,96 @@
+            "e": "AQAB",
+            "x5c": [
+                "MIIDUjCCAjqgAwIBAgIUX2ap2e4rmTrPAQM/A4P6ojCg+5gwDQYJKoZIhvcNAQEFBQAwNDELMAkGA1UEBhMCVVMxDzANBgNVBAoMBlNQSUZGRTEUMBIGA1UEAwwLZXhhbXBsZS5vcmcwIBcNMjEwNTI3MTQ0NDM5WhgPMjEyMTA1MDMxNDQ0MzlaMDQxCzAJBgNVBAYTAlVTMQ8wDQYDVQQKDAZTUElGRkUxFDASBgNVBAMMC2V4YW1wbGUub3JnMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0mfowcVtUL+Qkf8ommz+afivIXJiV1emoZ199b1CDXEiGR5/ordFvz6y4l8/q6i0psKIhYaag56nTUSk5kjcUJpzuVkVV5vdG3IVLo4n3ULbZEjZEnorWlhYJcAJLNJhYk6An32uz4+pjlzigSxghinUBFG8e5A+etAST6VZ8HtIWErjEe92R8paB56IACwzbhFpJ/OTQu6FJNiRf5XDipMf6whXFwmwEws+FiZa+m5+4zE+QSWxS1WmJM4qAJsCcCJDvuc7/6JoF8Y7O6wsIVZ0vvg1f26caXG80bYBSF/xX6gcSgFa0OvrMauVNFIbVKZkdOgdjftOPvni7OpIUQIDAQABo1owWDAJBgNVHRMEAjAAMB8GA1UdEQQYMBaGFHNwaWZmZTovL2V4YW1wbGUuY29tMB0GA1UdDgQWBBRZCWP9Q5e/8cQ3xwDtHpzZrMVCkDALBgNVHQ8EBAMCB4AwDQYJKoZIhvcNAQEFBQADggEBAAJOA81nY/8ItGUO2BZey6wnH88fnTwmum2XG/uzUeYIuGMztUuoGzQ5bpOVhh8Fe9O3biJ1u+zJzRWO6J4XuMWkjGCGp6oYvnwXxvd6nas1HkqEH7q/4hgX4CtHUAWnW5sa8t2jbG+a5x7534ZZQ5VwMZXTm07wAMtRVTqRYl6nSHUWCDowZjqR02tJRtGbSia04tq17ojcdIa7mVrrfFU1Frn3kCh6ifyIk9lOVhhR3B4evyOuanLPUjj8QHR7yPTxtDsBQ+FGwhC9W7hAaMcu7k8iy8ASaqM2w4bCUPpiQWm6kVeHUbr7jEoB/Ym87fInIorVFysctktK2pSlcgU="
+            ]
+        }
+    ],
+    "spiffe_refresh_hint": 60
+}
+`
+
+type fakeSource struct {
+	bundles map[spiffeid.TrustDomain]*spiffebundle.Bundle
+}
+
+func (s *fakeSource) GetBundleForTrustDomain(trustDomain spiffeid.TrustDomain) (*spiffebundle.Bundle, error) {
+	b, ok := s.bundles[trustDomain]
+	if !ok {
+		return nil, errors.New("bundle not found")
+	}
+	return b, nil
+}
+
+func main() {
+
+	svid, err := x509svid.Load("./resources/example.org.crt", "./resources/example.org.key")
+	if err != nil {
+		log.Fatal(err)
+	}
+	cert := &tls.Certificate{
+		Certificate: make([][]byte, 0, len(svid.Certificates)),
+		PrivateKey:  svid.PrivateKey,
+	}
+	for _, svidCert := range svid.Certificates {
+		cert.Certificate = append(cert.Certificate, svidCert.Raw)
+	}
+
+	trustDomain, _ := spiffeid.TrustDomainFromString("example.org")
+	bundle, _ := spiffebundle.Parse(trustDomain, []byte(jwks))
+	source := &fakeSource{}
+	source.bundles = map[spiffeid.TrustDomain]*spiffebundle.Bundle{
+		trustDomain: bundle,
+	}
+	writer := new(bytes.Buffer)
+	handler := federation.Handler(trustDomain, source, logger.Writer(writer))
+	// create a custom server with `TLSConfig`
+
+	s := &http.Server{
+		Addr:    "example.org:443",
+		Handler: handler, // use `http.DefaultServeMux`
+		TLSConfig: &tls.Config{
+			Certificates: []tls.Certificate{*cert},
+			// GetCertificate: func(*tls.ClientHelloInfo) (*tls.Certificate, error) {
+			// 	return cert, nil
+			// },
```

Comment:
Shouldn't it be de-commented or removed?

Reply:
fair enough!